### ISSUE TITLE
Add mpv player, decode on Android player side, load ass/ssa subtitles

### ISSUE THREAD

**Issue by youtehub** [jellyfin/jellyfin-android#1489]
**Describe the feature you'd like**

I hope to support mpv player, so that the player can decode it. As shown in the picture below, the exo player currently used needs to decode the server before playing when loading ass/ssa subtitles. I hope to support mpv player, so that the decoding and loading of all subtitles can be done directly on the Android side, instead of relying on the server transcoding before playing.

**Additional context**

![862dcb800d3beb0a169e10f031e8df2c](https://github.com/user-attachments/assets/8c30d044-d0f2-4ad2-9fcb-08184f48ad04)

![86ddca5ce05bcb3e3b525c2934bc7702](https://github.com/user-attachments/assets/7f4ee7f1-6696-4b51-b9e2-1652238e08e9)

**Comment by 0x90shell** [comment]
I agree I'd like something more robust added as the internal player. Since the removal of libVLC half my library cannot play, native playback not transcoding, on the TV app.

**Comment by j-krl** [comment]
@youtehub Are you currently implementing this yourself? I'd love to see this. I moved to Findroid because of the playback tracking issues with external players, as Findroid supports MPV natively. Another feature that MPV adds is seek gestures.

**Comment by peerless2012** [comment]
> 我同意，我希望添加更强大的内置播放器。自从删除 libVLC 以来，我的一半库都无法在电视应用程序上播放，无法进行原生播放，无法进行转码。

Due to ass subtitles?

**Comment by Roboatlas21** [comment]
I find mpv player (streamyfin/findroid) at least falls back to hdr10 when direct playing dolby vision profile 7.6 files, exoplayer (official jellyfin app) on my s23 ultra just goes to black screen.
